chevah_build

```diff
@@ -195,7 +199,7 @@ install_dependencies() {
             install_command='sudo zypper --non-interactive install -l'
             check_command='rpm --query'
         ;;
-        *)
+        linux|aix*|solaris*|osx*|windows*)
```

Comment:
Is this a bad merge with master? I replaced the ever-changing string with `*` a while ago, just so that we don't have to update it every time we add an OS without package management. So, if it's not ubuntu, raspbian, rhel or sles, no package management, no need to list all the others.